src/main/kotlin/io/meshcloud/dockerosb/persistence/GitHandler.kt

```diff
@@ -7,30 +7,33 @@ import org.eclipse.jgit.api.Git
 import org.eclipse.jgit.transport.SshSessionFactory
 import org.eclipse.jgit.transport.URIish
 import org.eclipse.jgit.transport.UsernamePasswordCredentialsProvider
-import org.springframework.stereotype.Service
 import java.io.File
 
-@Service
-class GitHandler(
-    private val gitConfig: GitConfig
+open class GitHandler(
+  private val gitConfig: GitConfig
 ) {
 
-  fun commitAndPushChanges(
-      filePaths: List<String>,
-      commitMessage: String
+  open fun commit(
```

Comment:
At the moment yes. I will document that.
I think we can make it multi-node, the only problem to solve is i think the update of instances then, because this should be the only source for conflicts.